account-admin/byob.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "Bring Your Own S3 Bucket"
+description: "Configure your own AWS S3 bucket to work with Rive"
+---
+
+## What is a Custom S3 Bucket?
+
+A custom S3 bucket allows Enterprise customers to store their Rive assets in their own AWS environment instead of Rive's default storage. This premium feature gives you greater control over your data, allowing you to:
+
+- Apply your organization's security and compliance policies
+- Keep data within your existing AWS infrastructure
+- Maintain direct ownership of storage costs and capacity
+- Implement custom lifecycle policies for data retention
+- Integrate with your existing backup and disaster recovery processes
+- Monitor and audit all access to your bucket using CloudTrail
+
+**Note:** This feature is available exclusively to Enterprise plan customers. If you're interested in upgrading to unlock this capability, [contact us](https://rive.app/enterprise).
+
+If you have existing data in Rive that you would like migrated to your custom S3 bucket after upgrading to Enterprise, our team can help coordinate the migration process.
+
+## Configuration
+
+<Steps>
+  <Step title="Create an S3 bucket in your AWS Account">
+    Follow the [AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to create a new S3 bucket:
+    
+    - Choose a unique name for your bucket <br />
+      **Note:** Use that name in place of `BUCKET_NAME` for the rest of this document
+    - Configure basic bucket settings
+      - Block public access
+      - You can choose to leave everything as default or you can decide to enable or customize:
+        - versioning (disabled by default)
```

Comment:
I wouldn't get them to turn this on... It will massively increase costs based on how we write over revisions all the time